modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -76,7 +88,7 @@ const renderInlineDialog = <T extends Dialog.DialogData>(dialogInit: DialogManag
   const dialog = GuiFactory.build({
     dom: {
       tag: 'div',
-      classes: [ 'tox-dialog', inlineClass ],
+      classes: [ 'tox-dialog', inlineClass, ...dialogSize ],
```

Comment:
```suggestion
      classes: [ 'tox-dialog', inlineClass, ...dialogSize.toArray() ],
```
Follow on from previous comment.